packages/core/src/common/util/migration/migrations/migrateToV10.ts

```diff
@@ -0,0 +1,8 @@
+import { experimentSchema } from '../../../types/common'
+
+export const migrateToV10 = (json: any) => {
```

Comment:
Bliver det træls med 'any' i migrate-funktionerne?

Reply:
Det kan man godt. Grunden til at fjerne Vx typerne er blot for at rydde op. De gamle migrationer skal jo i princippet ikke røres igen.